cfme/tests/automate/custom_button/test_container_objects.py

```diff
@@ -0,0 +1,106 @@
+import pytest
+import fauxfactory
+
+from widgetastic_patternfly import Dropdown
+
+from cfme.containers.provider import ContainersProvider
+from cfme.markers.env_markers.provider import ONE_PER_TYPE
+from cfme.utils.appliance.implementations.ui import navigate_to
+from cfme.utils.log import logger
+
+
+pytestmark = [
+    pytest.mark.tier(2),
+    pytest.mark.usefixtures("setup_provider"),
+    pytest.mark.provider([ContainersProvider], selector=ONE_PER_TYPE),
+]
+
+CONTAINER_OBJECTS = [
+    "CONTAINER_IMAGE",
+    "CONTAINER_NODE",
+    "CONTAINER_POD",
+    "CONTAINER_PROJECT",
+    "CONTAINER_TEMPLATE",
+    "CONTAINER_VOLUME",
+]
+
+DISPLAY_NAV = {
+    "Single entity": ["Details"],
+    "List": ["All"],
+    "Single and list": ["All", "Details"],
+}
+
+
+@pytest.fixture(
+    params=CONTAINER_OBJECTS, ids=[obj.capitalize() for obj in CONTAINER_OBJECTS], scope="module"
```

Comment:
It seems like capitalize is not needed any more.

Reply:
@jaryn we won't take ids in full capital letters. So its required